tests/pytest/ztp-left-shifting/sno-hub/check-hub-installation.py

```diff
@@ -0,0 +1,45 @@
+import os
+import time
+import requests
+import pytest
+
+kubeconfig=os.getenv("KUBECONFIG")
+ocp_hub_version=os.getenv("OCP_HUB_VERSION")
+mch_ns=os.getenv("MCH_NAMESPACE")
```

Comment:
I think that according to PEP8 we should have space in the equality sign
```suggestion
kubeconfig = os.getenv("KUBECONFIG")
ocp_hub_version = os.getenv("OCP_HUB_VERSION")
mch_ns = os.getenv("MCH_NAMESPACE")
```